templates/form/input--submit.html.twig

```diff
@@ -11,12 +11,18 @@
  */
 #}
 
-{# Header search sumbit #}
+{# Header or Directories search submit #}
 {% set form_input_submit = 'header-search-block-form-submit' %}
+{% set is_dir_channel = attributes.hasAttribute('data-drupal-selector') and attributes['data-drupal-selector']|render starts with 'edit-submit-localgov-directory-channel' %}
 
-{# Changing the header's search submit input element to a button element #}
-{% if form_input_submit in attributes.class %}
+{# Changing the header or directory search submit input element to a button element #}
+
+{% if (form_input_submit in attributes.class) %}
     <button class="btn icon-search" type="submit" value="Search" aria-label="Search this website" title="Search this website"><span class="fa fa-search" aria-hidden="true"></span></button>
+{% elseif (is_dir_channel)%}
```

Comment:
[if statements in Twig](https://twig.symfony.com/doc/2.x/tags/if.html) can do without the brackets around the conditions.  This means `{% if (form_input_submit in attributes.class) %}` is the equivalent of the simpler `{% if form_input_submit in attributes.class %}`

Reply:
fixed